deps/src/main/java/com/microsoft/azure/sdk/iot/deps/twin/TwinCollection.java

```diff
@@ -224,37 +207,30 @@ public final void putAllFinal(Map<? extends String, ?> map)
      * <p> Override {@code HashMap.put(String, Object)}.
      *
      * <p> This function will add a single pair key value to the TwinCollection. By the
-     *     Twin definition, the {@code Object} can contain types of {@code Boolean},
-     *     {@code Number}, {@code String}, {@code Object}, or up to 5 levels of
-     *     sub-TwinCollection, but it cannot be types defined by the user or arrays.
+     * Twin definition, the {@code Object} can contain types of {@code Boolean},
+     * {@code Number}, {@code String}, {@code Object}, or up to 5 levels of
+     * sub-TwinCollection, but it cannot be types defined by the user or arrays.
      *
-     * @param key the {@code String} that represent the key of the new entry. It cannot be {#code null} or empty.
+     * @param key   the {@code String} that represent the key of the new entry. It cannot be {#code null} or empty.
```

Comment:
Not your fault, but I think this is meant to be `{@code null}`

Also, should be "represents"